development-updates.md

```diff
@@ -122,7 +122,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
 | [Pat Stiles](https://github.com/PatStiles)                     | [Update 3](https://hackmd.io/@ZZhqXtsPTHawMbCDb4ZkaA/HJj4cBrjh)                                                                      |                                                                                                               |                                                                                                                                                                           |
 | [Rado](https://github.com/radkomih/)                           |                                                                                                                                      |                                                                                                               |                                                                                                                                                                           |
 | [Roy](https://github.com/roycncn/)                             |                                                                                                                                      |                                                                                                               |                                                                                                                                                                           |
-| [Scar Face](https://github.com/scarfacedotcom)                 | [Update 3](https://hackmd.io/@0xScarFace/epfweek3)                                                                                   | [Update 4](https://hackmd.io/@0xScarFace/epfweek4b)                                                           |                                                                                                                                                                           |
+| [Scar Face](https://github.com/scarfacedotcom)                 | [Update 3](https://hackmd.io/@0xScarFace/epfweek3)                                                                                   | [Update 4](https://hackmd.io/@0xScarFace/epfweek4b)                                                           |  [Fuzzing Ethereum’s devp2p Protocol](https://hackmd.io/@0xScarFace/projectproposal)                                                                                                                                                                         |
```

Comment:
The link to project proposal should point to a document within repository

Reply:
Done

Reply:
Done

Reply:
Done